lib/scripts.js

```diff
@@ -329,15 +312,6 @@ var scripts = {
     return keys.concat([pushCmd, jobId]);
   },
 
-  retryJob: function(job){
```

Comment:
This is unused, and has a reference error (`jobId` doesn't exist). So I'm just removing it.